pkg/kubelet/status/manager.go

```diff
@@ -78,6 +78,8 @@ type Manager interface {
 	// SetPodStatus caches updates the cached status for the given pod, and triggers a status update.
 	SetPodStatus(pod *api.Pod, status api.PodStatus)
 
+	DeletePodStatus(uid types.UID)
```

Comment:
In the future, I'd like to see Status being integrated into the Runtime interface completely.